core/unix/os.c

```diff
@@ -347,6 +347,16 @@ static byte *app_brk_cur;
 static byte *app_brk_end;
 #endif
 
+#ifdef MACOS
+static int macos_version;
+# define MACOS_VERSION_SIERRA 16
+# define MACOS_VERSION_EL_CAPITAN 15
+# define MACOS_VERSION_YOSEMITE 14
+# define MACOS_VERSION_MAVERICKS 13
+# define MACOS_VERSION_MOUNTAIN_LION 12
+# define MACOS_VERSION_LION 11
```

Comment:
(Looks good static in this .c for now, but in the future we might want to consider exposing these via the dr_get_os_version() API.)